AnnoyancesFilter/sections/widgets.txt

```diff
@@ -305,6 +305,7 @@ amp.focus.de##amp-embed[type="opinary"]
 ! Chat widgets
 !
 moretvtime.com##.cov
+kaspersky.de###nina-kaspersky
```

Comment:
this is not third-party widget. Not sure if we need to block it.

Reply:
@BlazDT @AdamWr

Reply:
should we block it as popup/floating element? @Alex-302

Reply:
As it is not a third-party chat widget, I think it is better not to block it.

Reply:
It's pretty questionable to block it. @Sergey-Lyapin please remove the rule from pr.